packages/core/contracts/financial-templates/funding-rate-store/test/MockFundingRateStore.sol

```diff
@@ -41,6 +41,9 @@ contract MockFundingRateStore is FundingRateStoreInterface, Testable {
         view
         returns (FixedPoint.Unsigned memory)
     {
+        if (fundingRates[identifier].length == 0) {
+            return FixedPoint.fromUnscaledUint(1);
```

Comment:
I like this. Maybe on construction we just set the first one to 0 or set a default